tests/msc2716_test.go

```diff
@@ -305,6 +305,47 @@ func TestImportHistoricalMessages(t *testing.T) {
 			}
 		})
 
+		t.Run("Non-member state events are allowed in state_events_at_start", func(t *testing.T) {
+			t.Parallel()
+
+			roomID := as.CreateRoom(t, createPublicRoomOpts)
+			alice.JoinRoom(t, roomID, nil)
+
+			// Create the "live" event we are going to insert our historical events next to
+			eventIDsBefore := createMessagesInRoom(t, alice, roomID, 1, "eventIDsBefore")
+			eventIdBefore := eventIDsBefore[0]
+			timeAfterEventBefore := time.Now()
+
+			state_events_at_start := createJoinStateEventsForBatchSendRequest([]string{virtualUserID}, timeAfterEventBefore)
```

Comment:
`camelCase` please.

Reply:
Is there a linter to use to avoid these kinds of mistakes?

Reply:
There is, but I don't think it's built into CI. Added it to https://github.com/matrix-org/complement/issues/290